src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,312 @@
+import {
+  Box,
+  Flex,
+  Text,
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuSeparator,
+  DropdownMenuItem,
+  Button,
+} from "@sparrowengg/twigs-react";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverArrow,
+  PopoverContent,
+} from "@sparrowengg/twigs-react";
+
+import { CalendarIcon } from "../Components/Icons";
+
+import LeftArrow from "./../asset/leftArrow.svg";
+import moonIconDashboard from "./../asset/moonSvgDemo.svg";
+import Plus from "./../asset/plus.svg";
+import plusLight from "./../asset/plusLight2.svg";
+import AlertCircle from "./../asset/alert-circle.svg";
+import AlertCircle2 from "./../asset/alert-circle2.svg";
+import HorizontalDots from "./../asset/more-horizontal.svg";
+import HorizontalDotsDark from "./../asset/more-horizontalDark.svg";
```

Comment:
@9xVibee this naming doesn't seems relevant can you change the casing and the name for it

Reply:
Regarding names could please give a example for it!